packages/errors/package.json

```diff
@@ -16,18 +16,18 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "lib/index.js",
-  "module": "lib-es/index.js",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
```

Comment:
these shouldn't be the same folder right? or maybe i miss something. 

my understanding is: essentially the "module" helps optimizing bundle size and should be the new way of doing things, main is here for backward compat.

Reply:
I bundle the file only in umd format so I thought it works for both platform. But if you want, I can change it so that the bundler outputs several files with different format.

Reply:
preferably yes. this was a community request that we addressed recently https://github.com/LedgerHQ/ledgerjs/issues/397

Reply:
I can't denied request from a community especially from those two 😄
fix: https://github.com/LedgerHQ/ledgerjs/pull/467/commits/04f9a934c368a8ce35e5635d20dfb7a4a8862873